### ISSUE TITLE
Citation

### ISSUE THREAD

**Issue by Hossam-Tarek33** [open-mmlab/mmcv#1159]
How can i cite mmcv ?? is there any bibtex citation available ??
Thanks in advance

**Comment by ZwwWayne** [comment]
Hi @Hossam-Tarek33 ,

Yes, you can. The bibtex is as below:

```
@misc{mmcv,
    title={{MMCV: OpenMMLab} Computer Vision Foundation},
    author={MMCV Contributors},
    howpublished = {\url{https://github.com/open-mmlab/mmcv}},
    year={2018}
}

```

**Comment by zhouzaida** [comment]
resolved by #1160
